src/Runner.Common/Tracing.cs

```diff
@@ -27,6 +27,11 @@ public Tracing(string name, ISecretMasker secretMasker, SourceSwitch sourceSwitc
             }
 
             _traceSource.Listeners.Add(traceListener);
+
+            if (StringUtil.ConvertToBoolean(Environment.GetEnvironmentVariable(Constants.Variables.Agent.PrintLogToStdout)))
+            {
+                _traceSource.Listeners.Add(new StdoutTraceListener());
```

Comment:
can we make the caller decide whether we need to pass a `StdoutTraceListener` to `Tracing()`?
ex: 
`public Tracing(string name, ISecretMasker secretMasker, SourceSwitch sourceSwitch, HostTraceListener traceListener, StdoutTraceListener stdoutListener = null)`

Reply:
👍